### DISCUSSION TITLE
Unexpected Behavior of Histogram Metrics in Vector

### DISCUSSION THREAD

**Discussion by veer123prathap** [vectordotdev/vector#19311] [General]
I am using Vector for metric collection and have been observing an unexpected behavior with the Histogram metric type. Each time a new metric event is created, it appears to clear the old histogram metrics and adds the new metric event to the set of metrics. This behavior is unexpected as per my understanding of how Vector handles metrics.

In Vector, each time a new metric event is created, it should add the new metric event to the existing set of metrics, without clearing the old histogram metrics. However, I am observing that the old histogram metrics are being cleared each time a new metric is added.

This behavior is causing issues in my monitoring setup as I am losing the historical data of my histogram metrics. I have checked my Vector configuration and the documentation of the downstream service I am using, but I haven’t found a solution to this issue.

I am looking for guidance on why this behavior is occurring and how I can prevent the old histogram metrics from being cleared when a new metric is added. Any help would be greatly appreciated. Thank you.

**Comment by jszwedko** [comment]
This is a duplicate of https://github.com/vectordotdev/vector/issues/19304
